benchmarks/cpp/nvfuser/layer_norm.cpp

```diff
@@ -53,6 +53,68 @@ static void setupLayerNorm(Fusion* fusion, DataType dtype) {
   fusion->addOutput(output);
 }
 
+static void setupRMSNorm(Fusion* fusion, DataType dtype) {
```

Comment:
I've been putting functions in their own files, I think it'd be good to put RMS norm in an `rms_norm.cpp` and `rms_norm_bwd.cpp`